Plugin~/WebRTCPlugin/GraphicsDevice/Vulkan/VulkanGraphicsDevice.cpp

```diff
@@ -176,215 +227,246 @@ namespace webrtc
             VULKAN_API_CALL(vkResetFences(m_Instance.device, 1, &m_fence));
             VULKAN_API_CALL(vkResetCommandBuffer(m_commandBuffer, 0));
         }
+#else
+        VkCommandBuffer commandBuffer = texture->GetCommandBuffer();
+        if (!commandBuffer)
+            return;
+
+        VkResult result = vkEndCommandBuffer(commandBuffer);
+        if (result != VK_SUCCESS)
+        {
+            RTC_LOG(LS_ERROR) << "vkEndCommandBuffer failed. result:" << result;
+            return;
+        }
+
+        if (m_unityVulkan != nullptr)
+        {
+            m_unityVulkan->AccessQueue(AccessQueueCallback, 0, texture, false);
+        }
+        else
+        {
+            AccessQueueCallback(0, texture);
+        }
```

Comment:
We will need to swap this to Kazuki's updated version.
We didn't just revert the entire change, but also removed the AccessQueueCallback piece.